x/vesting/keeper/integration_test.go

```diff
@@ -166,38 +133,7 @@ var _ = Describe("Clawback Vesting Accounts", Ordered, func() {
 		})
 
 		It("can delegate vested tokens", func() {
-			_, err := s.app.StakingKeeper.Delegate(
-				s.ctx,
-				addr,
-				vested.AmountOf(stakeDenom),
-				stakingtypes.Unbonded,
-				s.validator,
-				true,
-			)
-			Expect(err).To(BeNil())
-		})
-
-		It("can vote on governance proposals", func() {
-			// Submit governance porposal
-			TestProposal := govtypes.NewTextProposal("Test", "description")
-			depositor := sdk.AccAddress(tests.GenerateAddress().Bytes())
-			proposalCoins := sdk.NewCoins(sdk.NewCoin(stakeDenom, s.app.StakingKeeper.TokensFromConsensusPower(s.ctx, 10)))
-			err := testutil.FundAccount(s.app.BankKeeper, s.ctx, depositor, proposalCoins)
-			s.Require().NoError(err)
-
-			proposal, err := s.app.GovKeeper.SubmitProposal(s.ctx, TestProposal)
-			s.Require().NoError(err)
-
-			_, err = s.app.GovKeeper.AddDeposit(s.ctx, proposal.ProposalId, depositor, proposalCoins)
-			s.Require().NoError(err)
-
-			// Vote
-			err = s.app.GovKeeper.AddVote(
-				s.ctx,
-				proposal.ProposalId,
-				addr,
-				govtypes.NewNonSplitVoteOption(govtypes.OptionYes),
-			)
+			err := delegate(clawbackAccount, 1)
```

Comment:
@danburck,
Here, we are in context `after cliff and before lockup`. This means that all tokens are `locked` and some are `vested`. From what I read, I was expecting that you can only delegate tokens that are `vested` and `unlocked`. Did I misunderstand?

Reply:
@loredanacirstea It's really important that we get this right, so thanks for challenging this. The way it was merged is correct. It is possible to delegate vested tokens (unlocked and locked).

Reply:
I discussed this last with @khoslaventures. I see that @fedekunze comment on the linear is proposing something else. 

@khoslaventures @fedekunze can you confirm that this is the expected behaviour?